sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatterSuite.scala

```diff
@@ -61,28 +61,36 @@ class CodeFormatterSuite extends SparkFunSuite {
         |  * line
         |  * comments
         |  */
+        |  // comment
         |
         |public function() {
         |/*comment*/
         |  /*comment_with_space*/
-        |code_body
+        |code_body /* comment */ code_body/**/code_body // comment
         |//comment
-        |code_body
+        |code_body /*
+        |  * multi
+        |  * line
+        |  * comments
+        |  */
         |  //comment_with_space
         |
-        |code_body
+        |code_body // comment
+        | /*
+        |  * multi
+        |  * line
+        |  * comments
+        |  */
         |}
       """.stripMargin
 
     val reducedCode = CodeFormatter.stripExtraNewLinesAndComments(code)
     assert(reducedCode ===
-      """
-        |public function() {
+      """public function() {
+        |code_body code_body code_body
         |code_body
         |code_body
-        |code_body
-        |}
-      """.stripMargin)
+        |}""".stripMargin)
```

Comment:
Can we add a new test unit instead of modifying the existing tests?

Reply:
ok